schema.graphql

```diff
@@ -224,6 +224,7 @@ type Solution @entity {
   solver: User
   feeReward: BigInt
   objectiveValue: BigInt
+  utility: BigInt
```

Comment:
nit; I think here we can have these be non-optional `BigInt!` properties. The reason I argue for this is that, while a Solution entity can exist without these values as trade events are being processed, there will never be empty at the end of processing a block.

Then, when the solution entity is created we can just use some default values:
```
  // Details will be populated onSolutionSubmission
  solution.solver = address(0)
  solution.feeReward = 0
  solution.utility = 0
  solution.objectiveValue = 0
```